UnityProject/Assets/Scripts/Objects/Directionals/Rotatable.cs

```diff
@@ -36,6 +36,10 @@ public enum RotationMethod
 	[SyncVar(hook = nameof(SyncServerLockAndDirection))]
 	private LockAndDirection SynchroniseCurrentLockAndDirection;
 
+	[SerializeField]
+	[Tooltip("If the direction is only handled by client")]
+	private bool updateFromServer = true;
```

Comment:
I don't understand this. Name of the variable suggests the server handles it, the tooltip says the contrary

Reply:
I'll update the tooltip